src/features/chat/ChatMessage.tsx

```diff
@@ -0,0 +1,66 @@
+import React /*, { CSSProperties }*/ from 'react'
+import { chatMsgDatetime } from '../../common/utils/format'
+import cn from 'classnames'
+import styles from './Chat.module.css'
+
+export const DIR_SENT = 0
+export const DIR_RECV = 1
+
+export interface ChatMessageProps {
+  text?: string
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
no-explicit-any is very important rule for code maintainability. So here I see `sender` prop, but I have no idea what it contains. Please don't use `any`